ajax/libs/aes-js/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "aes-js",
+  "version": "3.0.0",
+  "description": "A pure JavaScript implementation of the AES block cipher and all common modes of operation.",
+  "filename": "index.js",
```

Comment:
Can you also add a minified `index.min.js`?
Also change `"filename"` to `"index.min.js"`

Our recommend minify tool: https://github.com/PeterDaveHello/web-minify-helper